modules/nf-core/narfmap/align/tests/segfault.nf.test

```diff
@@ -0,0 +1,55 @@
+nextflow_process {
+
+    name "Test Process NARFMAP_ALIGN"
+    script "../main.nf"
+    process "NARFMAP_ALIGN"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "narfmap"
+    tag "narfmap/align"
+    tag "narfmap/hashtable"
+    tag "bug"
+
+    test("This should segfault") {
```

Comment:
```suggestion
    test("Human - bam") {
```

Reply:
This is a regression test, not a standard test. I don't know if it's related to the sample being Human or not. But I think it should break naming conventions.